src/components/dialog-manager.js

```diff
@@ -53,6 +53,10 @@ export class DialogManager {
    * @return {!Promise}
    */
   openView(view) {
+    view.acceptResult().catch(reason => {
+      this.completeView(view);
```

Comment:
For the time being, can we only close when "cancel", e.g. `reason.name == 'AbortError'`? It's useful for debugging to keep the view option for other failures.

Reply:
Done.